soc/arm/nxp_imx/rt/soc.c

```diff
@@ -208,22 +208,6 @@ static ALWAYS_INLINE void clkInit(void)
 
 }
 
-#if defined(CONFIG_DISK_ACCESS_NXP_USDHC1) ||	\
-	defined(CONFIG_DISK_ACCESS_NXP_USDHC2)
-u32_t usdhc_src_clk_freq(u16_t nusdhc)
-{
-	if (nusdhc == 0) {
-		return CLOCK_GetSysPfdFreq(kCLOCK_Pfd0) /
-				(CLOCK_GetDiv(kCLOCK_Usdhc1Div) + 1U);
-	} else if (nusdhc == 0) {
-		return CLOCK_GetSysPfdFreq(kCLOCK_Pfd0) /
-				(CLOCK_GetDiv(kCLOCK_Usdhc2Div) + 1U);
```

Comment:
You added this in a previous commit in the same PR, then removed it. You're right that this code should be in the clock_control driver, but please fix up the git history in the PR